src/wayland-window.cpp

```diff
@@ -110,7 +113,17 @@ namespace wf
         {
             gtk_layer_set_anchor(this->gobj(),
                 (GtkLayerShellEdge)layer_anchor, true);
-        }
+        } else if (layer_anchor == ANCHOR_PINNED_BOTTOM)
+	{
```

Comment:
Same here, use spaces and indent with the rest of the code.

Reply:
Done fd049b29455cb7d594f49d29f5292bb580aea9ea.